Tasks/HelmDeployV0/Tests/TestSetup.ts

```diff
@@ -97,6 +98,10 @@ const a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
 if (process.env[shared.TestEnvVars.command] === shared.Commands.install) {
     let helmInstallCommand = "helm install";
 
+    if (debug) {
+        helmInstallCommand = helmInstallCommand.concat(` --debug`);
```

Comment:
Could it make sense to put it to some method - to avoid duplication?

Reply:
Thanks. Applied change.